vips/govips.go

```diff
@@ -20,15 +15,6 @@ const (
 	defaultMaxCacheSize     = 500
 )
 
-// VipsVersion if the primary version of libvips
-const VipsVersion = string(C.VIPS_VERSION)
```

Comment:
Why remove the vips version constants? It's good that they're there

Reply:
fine, also added patch (micro) while at it